lib/rules/require-splattributes.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const Rule = require('./base');
+
+module.exports = class RequireSplattributes extends Rule {
+  visitor() {
+    let foundSplattributes = false;
+
+    return {
+      AttrNode(node) {
+        if (node.name === '...attributes') {
+          foundSplattributes = true;
+        }
+      },
+
+      Template: {
+        exit(node) {
+          if (!foundSplattributes) {
+            let { body } = node;
+            let elementNodes = body.filter((it) => it.type === 'ElementNode');
+            let nonEmptyTextNodes = body.filter((it) => it.type === 'TextNode' && it.chars.trim());
+            if (elementNodes.length === 1 && nonEmptyTextNodes.length === 0) {
+              this.report({
+                message: 'The root element in this template should use `...attributes`',
+                node: elementNodes[0],
+              });
+            } else {
+              this.report({
+                message: 'At least one element in this template should use `...attributes`',
+                node,
+              });
+            }
```

Comment:
Isn't this pretty odd for route templates? Where would you put `...attributes` there? 

Is your thought here that you'd only use this rule on the components subdirectory?

Reply:
indeed, I didn't think about this when I implemented it and I was only thinking about components 😅 

since our config supports `overrides` I suppose it would be possible though to only enable it for component files 🤔 

I suppose you would want me to mention that in the rule docs though?

Reply:
> I suppose you would want me to mention that in the rule docs though?

Ya, exactly.

Reply:
done :)